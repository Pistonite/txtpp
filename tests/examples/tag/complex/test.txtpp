TXTPP#tag tag1
-TXTPP#run echo "tag1 content"
TXTPP#tag tag2
injecting tag1...
-TXTPP#include include
TXTPP#tag tag3
injecting tag2...
-TXTPP#run echo "tag3 content"
injecting tag3...
end of file