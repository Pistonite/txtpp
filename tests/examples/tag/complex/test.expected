injecting tag1 content
...
injecting include content...
injecting tag3 content
...
end of file
